modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -81,37 +61,61 @@ const Card = ({item, schema}) => {
 						});
 					}
 				},
-				symbolLeft: action.icon,
 			}))}
 			description={localizedDescription}
 			href={(schema.link && item[schema.link]) || null}
-			imgProps={schema.image && item[schema.image]}
+			imgProps={imageProps}
 			onSelectChange={
-				selectable && (() => selectItems(item[selectedItemsKey]))
+				selectable
+					? () => {
+							selectItems(selectedItemKey);
+					  }
+					: undefined
 			}
-			selected={
-				selectable &&
-				!!selectedItemsValue.find(
-					(element) => element === item[selectedItemsKey]
-				)
-			}
-			stickerProps={schema.sticker && item[schema.sticker]}
+			selected={cardSelected}
+			stickerProps={null}
```

Comment:
A bit scary, we're basically removing API here. It's probably fine though, since nobody was using `Cards` view in DXP, until now. A move conservative change would be something like
```js
stickerProps={(schema.sticker && item[schema.sticker]) || null}
```

Reply:
Done!